tests/kafkatest/services/verifiable_producer.py

```diff
@@ -55,16 +80,28 @@ def _worker(self, idx, node):
                     elif data["name"] == "producer_send_success":
                         self.acked_values.append(int(data["value"]))
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-verifiable-producer.sh" \
+    def start_cmd(self, node):
+
+        cmd = ""
+        if node.version <= LATEST_0_8_2:
+            # 0.8.2.X releases do not have VerifiableProducer.java, so cheat and add
+            # the tools jar from trunk to the classpath
+            cmd += "for file in /opt/kafka-trunk/tools/build/libs/kafka-tools*.jar; do CLASSPATH=$CLASSPATH:$file; done; "
```

Comment:
Can we just do this in the provisioning script? Seems like this should really be a one-time setup step rather than running every time we use `VerifiableProducer`.